src/constants.ts

```diff
@@ -15,7 +15,8 @@ export const {
   REACT_APP_TOKEN_URL: TOKEN_URL,
   REACT_APP_REVOKE_URL: REVOKE_URL,
   REACT_APP_LOGIN_REDIRECT_URL: LOGIN_REDIRECT_URL,
-  REACT_APP_FHIR_API_ADMIN_TOKEN: FHIR_API_ADMIN_TOKEN
+  REACT_APP_FHIR_API_ADMIN_TOKEN: FHIR_API_ADMIN_TOKEN,
+  REACT_APP_FILES_SERVER_URL: FILES_SERVER_URL
```

Comment:
should this end with a trailing slash ? i'll add this in deployment

Reply:
yes it should!